### PR TITLE
Added portfolio for rerum admin work

### PR DESCRIPTION
**Pull request by alar12**
Updated portfolio.md and created a dedicated folder for rerum admin work

### CODE REVIEW

File: docs/portfolio.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ We are especially interested in projects supporting researchers, focused on rese
   
 **[Where's Religion Mobile](project_wheres_religion_mobile/about.md)** - This Project is to create an application for ethnographers to capture the events happening around them along with some audio/visual representation of the event.  
 
+**[Rerum Admin Work](project_rerum_admin_work/about.md)** - A tiny NodeJS web application for connecting with the RERUM API. This is for software developers looking to develop a client application that uses the public RERUM API as the client's back end.
```
Comment (kungfuchicken) [inline docs/portfolio.md:48]: Please move this to the External Projects section that @MutyaSowmya123 created. It's merged into main, so you can either merge main into your branch or rebase your branch off of main to get this section.

File: docs/project_rerum_admin_work/about.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+### **Architecture**
```
Comment (kungfuchicken) [inline docs/project_rerum_admin_work/about.md:66]: Please restructure this as a paragraph of text under a "Technical Overview" section to match the structure of other existing pages. See https://oss-slu.github.io/docs/project_pi4micronaut/about and https://oss-slu.github.io/docs/project_wheres_religion_mobile/about for examples of the sort of format. Consistency in this format is important for branding.

**Review by kungfuchicken — CHANGES_REQUESTED** [review]
making progress, and almost there.

File: docs/project_rerum_admin_work/about.md (hunk 2 above)
Comment (kungfuchicken) [reply docs/project_rerum_admin_work/about.md:66]: to be clear, the mermaid diagram is fine. i'm talking about the multiple bulleted lists from lines 66-91.

**Comment by kungfuchicken** [comment]
I don't want to lose this great writeup (you used Mermaid!!!), but since we're pausing on this I don't want to merge it, either. i will leave this open  until we actually start working on it.

**Comment by alar12** [comment]
> I don't want to lose this great writeup (you used Mermaid!!!), but since we're pausing on this I don't want to merge it, either. i will leave this open until we actually start working on it.

That’s okay professor we will deal with this later when we start working on it again.

**Comment by kungfuchicken** [comment]
I'm not sure if/when we'll start this project. I'm closing this for now. It's up to the next Tech Lead who works on it to start their own.

**Pull request closed without merging**